src/rpc/daemon_handler.h

```diff
@@ -132,7 +133,7 @@ class DaemonHandler : public RpcHandler
 
     void handle(const GetOutputDistribution::Request& req, GetOutputDistribution::Response& res);
 
-    std::string handle(const std::string& request);
+    epee::byte_slice handle(const std::string& request) override final;
```

Comment:
```suggestion
    epee::byte_slice handle(const std::string& request) final;
```

Reply:
So I just looked at this. I don't understand, do you think the `override` keyword should be dropped?

Reply:
Either way is fine to me